WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerAutomatticTracks.m

```diff
@@ -665,6 +665,24 @@ + (TracksEventPair *)eventPairForStat:(WPAnalyticsStat)stat
         case WPAnalyticsStatEnhancedSiteCreationSegmentsSelected:
             eventName = @"enhanced_site_creation_segments_selected";
             break;
+        case WPAnalyticsStatEnhancedSiteCreationSiteDesignViewed:
+            eventName = @"enhanced_site_creation_site_design_viewed";
+            break;
+        case WPAnalyticsStatEnhancedSiteCreationSiteDesignSelected:
+            eventName = @"enhanced_site_creation_site_design_selected";
+            break;
+        case WPAnalyticsStatEnhancedSiteCreationSiteDesignSkipped:
+            eventName = @"enhanced_site_creation_site_design_skipped";
+            break;
+        case WPAnalyticsStatEnhancedSiteCreationSiteDesignPreviewViewed:
+            eventName = @"enhanced_site_creation_site_design_preview_viewed";
+            break;
+        case WPAnalyticsStatEnhancedSiteCreationSiteDesignPreviewLoading:
+            eventName = @"enhanced_site_creation_site_design_preview_loading";
+            break;
+        case WPAnalyticsStatEnhancedSiteCreationSiteDesignPreviewLoaded:
+            eventName = @"enhanced_site_creation_site_design_preview_loaded";
+            break;
```

Comment:
I included the event names for the preview events in preparation for when we add that screen. This will just help prevent the need to create another library version